packages/components/FocusZone/macos/RCTFocusZone.m

```diff
@@ -480,6 +489,10 @@ - (void)keyDown:(NSEvent *)event
 		[[self window] makeFirstResponder:viewToFocus];
 		[viewToFocus scrollRectToVisible:[viewToFocus bounds]];
 	}
+	if (noOp == YES)
+	{
+		// No view to focus, do nothing
+	}
```

Comment:
Just wanted to double check that it was intentional for the else block below to be part of a different if statement? Also if so, could optionally do `else if (noOp == NO) ...`

Reply:
The newly added if clause should be inside of that block, so I'll update it